easybuild/easyblocks/i/imkl.py

```diff
@@ -112,7 +112,9 @@ def make_module_req_guess(self):
                     'FPATH': ['mkl/include', 'mkl/include/fftw'],
                 }
                 if LooseVersion(self.version) >= LooseVersion('11.0'):
-                    if LooseVersion(self.version) >= LooseVersion('11.1'):
+                    if LooseVersion(self.version) >= LooseVersion('11.3'):
+                        retdict['MIC_LD_LIBRARY_PATH'] = ['lib/intel64_lin_mic', 'mkl/lib/mic'];
```

Comment:
`mkl/lib/mic` is a symlink to `mkl/lib/intel64_lin_mic` in Intel MKL v11.3.x, so use the latter?

```
$ ls -ld imkl/11.3.*/mkl/lib/*mic
drwxr-xr-x 3 vsc40023 vsc40023 4096 Mar 17 11:46 imkl/11.3.0.109-iimpi-2016.00-GCC-4.9.3-2.25/mkl/lib/intel64_lin_mic
lrwxrwxrwx 1 vsc40023 vsc40023   17 Mar 17 11:45 imkl/11.3.0.109-iimpi-2016.00-GCC-4.9.3-2.25/mkl/lib/mic -> ./intel64_lin_mic
drwxr-xr-x 3 vsc40023 vsc40023 4096 Mar 17 11:52 imkl/11.3.1.150-iimpi-2016.01-GCC-4.9.3-2.25/mkl/lib/intel64_lin_mic
lrwxrwxrwx 1 vsc40023 vsc40023   17 Mar 17 11:52 imkl/11.3.1.150-iimpi-2016.01-GCC-4.9.3-2.25/mkl/lib/mic -> ./intel64_lin_mic
drwxr-xr-x 3 vsc40023 vsc40023 4096 Mar 17 11:55 imkl/11.3.2.181-iimpi-2016.02-GCC-4.9.3-2.25/mkl/lib/intel64_lin_mic
lrwxrwxrwx 1 vsc40023 vsc40023   17 Mar 17 11:54 imkl/11.3.2.181-iimpi-2016.02-GCC-4.9.3-2.25/mkl/lib/mic -> ./intel64_lin_mic
drwxr-xr-x 3 vsc40023 vsc40023 4096 May  8 07:58 imkl/11.3.3.210-iimpi-2016.03-GCC-5.3.0-2.26/mkl/lib/intel64_lin_mic
lrwxrwxrwx 1 vsc40023 vsc40023   17 May  8 07:57 imkl/11.3.3.210-iimpi-2016.03-GCC-5.3.0-2.26/mkl/lib/mic -> ./intel64_lin_mic
```